src/Components/Users/UsersContainer.tsx

```diff
@@ -1,7 +1,7 @@
 import React from "react";
 import { connect } from "react-redux";
 import { getCurrentPage, getFollow_unfollowUserIds, getIsFetching, getPageSize, getUsersCount, getUsersSuperReselector } from "../../Redux/user-selector";
-import { setTermAC, setIsFriendAC, follow, getUsers, setCurrentPage, setTotalUsersCount, setUsers, toggleIsFetching, toggleIsFollowingProgress, unfollow } from "../../Redux/users-reducer";
+import { actions, follow, getUsers, unfollow } from "../../Redux/users-reducer";
```

Comment:
If you decided to move most of the things to `actions`, what is the reason behind leaving `follow` and `unfollow` not under `actions`? Just trying to understand your logic here...

Reply:
follow, unfollow, getUsers are thunk creators

Reply:
Makes sense. This PR is just about moving things around, so I don't really have anything to comment on much. Feel free to merge it.